src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Defaulting and Validation of app CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic is enabled. This logic is provided by
+app-admission-controller.
+
+We have not enabled the defaulting and validation for existing app CRs to avoid
+disrupting your existing usage of App Platform.
+
+## Defaulting
+
+The defaulting logic is implemented using a mutating webhook. It currently
+defaults the following settings in the app CR if they are not specified.
+
+- The `app-operator.giantswarm.io/version` label that determines which instance of app-operator processes the app CR.
+- The kubeconfig secret that lets app-operator connect to your cluster.
+- The cluster values configmap that contains per cluster values like the base
+domain for your cluster.
```

Comment:
IMO it would be very useful to write down the actual default values (if possible) for these points

Reply:
Good idea. They will differ because the names have the cluster ID. But I'll add examples.